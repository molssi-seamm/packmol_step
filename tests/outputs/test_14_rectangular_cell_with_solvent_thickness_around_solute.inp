tolerance 2.0
output packmol.pdb
filetype pdb
connect yes
structure input_1.pdb
   inside cube 1.0000 1.0000 1.0000 17.0486
   center
   fixed 8.4779 8.5009 8.3967 0.0 0.0 0.0
   number 1
end structure
structure input_2.pdb
   inside cube 1.0000 1.0000 1.0000 17.0486
   number 222
end structure
