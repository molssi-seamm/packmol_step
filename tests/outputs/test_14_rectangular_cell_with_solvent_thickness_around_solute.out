    Will create a cubic periodic cell containing the following molecules:

        +-------------+------------------+---------+
        |  Component  | Structure        |   Ratio |
        |-------------+------------------+---------|
        |   solute    | c1ccccc1c2ccccc2 |         |
        |    fluid    | O                |       1 |
        +-------------+------------------+---------+

    The dimensions of the region will be calculated from the solute dimensions.
    If the input structure is periodic, its dimensions will be used. Otherwise
    for molecules, a the region will be a box with extra space of 10.0 Å around
    the molecule. The number of molecules of the fluid will be obtained  by
    using the density 1.0 g/ml.

Created a periodic cubic cell 19.05 Å on a side with the solute and 222.0 solvent molecules

    +-------------+------------------+---------------+----------+------------+
    |  Component  | Structure        |   Requested % |   Number | Actual %   |
    |-------------+------------------+---------------+----------+------------|
    |   solute    | c1ccccc1c2ccccc2 |               |        1 |            |
    |    fluid    | O                |       100.000 |      222 | 100.000    |
    +-------------+------------------+---------------+----------+------------+

There are a total of 688 atoms in the cell giving a density of 0.99791 g/ml.